content/en/flux/installation/bootstrap/gitea.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Flux bootstrap for Gitea
+linkTitle: Gitea
+description: "How to bootstrap Flux with Gitea"
+weight: 20
+---
+
+The [flux bootstrap gitea](/flux/cmd/flux_bootstrap_gitea/) command deploys the Flux controllers
+on a Kubernetes cluster and configures the controllers to sync the cluster state from a Gitea repository.
+Besides installing the controllers, the bootstrap command pushes the Flux manifests to the Gitea repository
+and configures Flux to update itself from Gitea.
+
+After running the bootstrap command, any operation on the cluster (including Flux upgrades)
+can be done via Git push, without the need to connect to the Kubernetes cluster.
+
+{{% alert color="danger" title="Required permissions" %}}
+To bootstrap Flux, the person running the command must have **cluster admin rights** for the target Kubernetes cluster.
+It is also required that the person running the command to be the **owner** of the Gitea repository,
+or to have admin rights of a Gitea organization.
+{{% /alert %}}
+
+## Gitea PAT 
+
+For accessing the Gitea API, the boostrap command requires a Gitea personal access token (PAT)
+with administration permissions.
+
+The Gitea PAT can be exported as an environment variable:
+
+```sh
+export GITEA_TOKEN=<gt-token>
+```
+
+If the `GITEA_TOKEN` env var is not set, the bootstrap command will prompt you to type it the token.
+
+You can also supply the token using a pipe e.g. `echo "<gt-token>" | flux bootstrap gitea`.
+
+## Gitea Personal Account
+
+If you want to bootstrap Flux for a repository owned by a personal account, you can generate a
+[Gitea PAT](https://gitea.com/user/settings/applications)
+that can create repositories by checking all permissions under `Select permissions` drop down menu.
+
+If you want to use an existing repository, the PAT's user must have `admin`
+[permissions](https://docs.gitea.com/development/oauth2-provider#scopes).
```

Comment:
Please provide more details on how to create the token with this permission. I don't see anything like "admin" in the UI for creating a token.

Reply:
Look like it's called `all` not admin, https://github.com/fluxcd/go-git-providers/blob/1f17f86ce0f5e1899185b59d5d0347c208ba80ec/tests/gitea/await-healthy.sh#L15C3-L15C3